scripts/eslint/index.js

```diff
@@ -11,9 +11,22 @@ const CLIEngine = require('eslint').CLIEngine;
 const cli = new CLIEngine();
 const formatter = cli.getFormatter();
 
-module.exports = function lintOnFiles(filePatterns) {
+function lintOnFiles(filePatterns) {
   const report = cli.executeOnFiles(filePatterns);
   const formattedResults = formatter(report.results);
   console.log(formattedResults);
   return report;
+}
+
+function validWarnings(report) {
```

Comment:
Let's move this to where it's used?
Also let's rename to something like `getSourceCodeWarnings`.

In fact I would just inline this logic in the script that uses it, and add a comment explaining why it exists.

Reply:
Changes reflect this. Do you prefer to have function declaration vs definition?